style.css

```diff
@@ -15,7 +15,7 @@ h1 {
   margin-left: 20px;
   color: #172b4d;
   font-style: normal;
-  font-weight: bolder;
+  font-weight: 900px;
```

Comment:
- [ ] Font-weights do not accept `px`. Please take a look at [this](https://www.w3schools.com/cssref/pr_font_weight.asp) article and fix your attribute value.(setting `font-weight: bolder;` was correct)